src/slices/main/sidebar/Sidebar.js

```diff
@@ -7,17 +7,24 @@ import styles from './Sidebar.css';
 class Sidebar extends Component {
     static propTypes = {
         className: PropTypes.string,
+        onAnimationEnd: PropTypes.func.isRequired,
```

Comment:
Perhaps move the CSS transition to `Sidebar.css` and add a `in` react property. The `onAnimationEnd` is called both on fadein/fadeout but with a boolean argument indicating its final state: visible/not-visible.